### DISCUSSION TITLE
Rename Product to Application in the Application Lifecycle event

### DISCUSSION THREAD

**Discussion by maxhotta** [ocsf/ocsf-schema#586] [Proposals]
There may be some confusion in the naming of the product attribute in the application lifecycle event, especially when the description uses the term "application":
"The application that was affected by the lifecycle event. This also applies to self-updating application systems."

The term application seems to make more sense in this context (e.g, application x has been installed).
I'd like to gather feedback whether a change should be made to rename the attribute from "product" to "app".
The change would be to add an additional "app" definition in the dictionary, where the type is "product", then include the "app" attribute in the lifecycle class instead of product.

**Comment by pagbabian-splunk** [comment]
Change was implemented.
